plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryUtil.java

```diff
@@ -83,6 +85,11 @@ public static String quote(String name)
         return QUOTE + name.replace(QUOTE, ESCAPED_QUOTE) + QUOTE;
     }
 
+    public static String quoteCommaSeparateValues(Collection<String> values)
```

Comment:
I would inline this method. We have a `quote` method using backtick ``` ` ``` in this class. Adding a similar method using different sign `'` will be little confusing.